user-operator/src/main/java/io/strimzi/operator/user/UserOperatorConfig.java

```diff
@@ -124,22 +125,20 @@ public static UserOperatorConfig fromMap(Map<String, String> map) {
     }
 
     public static int getClientsCaValidityDays() {
-        return getIntProperty(UserOperatorConfig.STRIMZI_CLIENTS_CA_VALIDITY);
+        return getIntProperty(UserOperatorConfig.STRIMZI_CLIENTS_CA_VALIDITY, AbstractModel.DEFAULT_CERTS_VALIDITY_DAYS);
     }
 
     public static int getClientsCaRenewalDays() {
-        return getIntProperty(UserOperatorConfig.STRIMZI_CLIENTS_CA_RENEWAL);
+        return getIntProperty(UserOperatorConfig.STRIMZI_CLIENTS_CA_RENEWAL, AbstractModel.DEFAULT_CERTS_RENEWAL_DAYS);
```

Comment:
Hmm. Having to add the CO as a dependency to access these constants is less than ideal. I suppose the argument could be made that these defaults are part of the CRD API, and so they could be exposed on `KafkaResources`, that would allows you to avoid needing the dependency.

Reply:
So I should move them to `io/strimzi/api/kafka/model/CertificateAuthority.java`?